news/3830.hooks.rst

```diff
@@ -0,0 +1,3 @@
+(OSX) When hook tkinter need to be loaded, this change fixes
+the inclusion of Tcl/Tk framework with python.org versions 
+coming bundled with its own.
```

Comment:
```suggestion
(OS X): If Python comes with a bundled Tcl/Tk framework, as recent versions
from python.org do, use this instead of the system one.
```